### PR TITLE
Breaking Change: Remove origin url and timestamp from deployments

### LINKED ISSUE #273: Stop exposing origin information
We currently support/expose the `originServerUrl` and `originTimestamp` because it was needed to keep compatibility with the old/history endpoint. Once we move into a more decentralized approach, we won't know where deployments were originated. The idea is to start removing that information from the `/deployments` endpoint.

We should:
- Stop exposing `originTimestamp` & `originServerUrl` on the `/deployments` payload

### PR DESCRIPTION
**Pull request by jmoguilevsky**
This PR removes the origin timestamp and origin server url from the deployments table and the corresponding endpoints responses.

Changes:
- <b> \* Breaking change \*</b> Add DB migration script to remove the correct columns and indexes in `deployments` and `failed_deployments` 
- Remove `origin_server_url` and `origin_timestamp` from the repository queries for deployments
- Remove `origin_server_url` and `origin_timestamp` from the repository queries for failed_deployments
- Add doc on how to run tests 
- Add doc on how to connect to db locally
- Update `dcl-catalyst-client` version to `5.0.1`

Fixes #273

### CODE REVIEW

File: content/README.md [hunk 1]
```diff
@@ -38,3 +42,9 @@ These are some of the more important configuration values:
 | STORAGE_ROOT_FOLDER | Folder where all content will be stored | 'storage' |
 | SERVER_PORT | Port to be used by the service | 6969 |
 | LOG_LEVEL | Minimum log level | 'info' |
+
+## Run unit tests
+    `yarn bazel run content:unit_test`
+
+## Run integration tests
```
Comment (agusaldasoro) [inline content/README.md:49]: Great!!

File: content/src/migrations/scripts/1620070748842_remove_origin.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+
+/*
+  In this migration, we are removing the origin_server_url and origin_timestamp colums
+ */
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.sql(`DROP INDEX IF EXISTS deployments_origin_timestamp_idx;`)
```
Comment (agusaldasoro) [inline content/src/migrations/scripts/1620070748842_remove_origin.ts:8]: Can you check if this is necessary or just removing the column is enough?

File: content/src/migrations/scripts/1620070748842_remove_origin.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+export async function down(pgm: MigrationBuilder): Promise<void> {
+  pgm.sql(`CREATE INDEX ON deployments ( origin_timestamp DESC );`) // Using plain SQL since lib doesn't expose DESC
+
+  pgm.sql(
+    `ALTER TABLE deployments ADD COLUMN origin_server_url text  DEFAULT 'https://peer.decentraland.org/content' NOT NULL;`
+  )
+  pgm.sql(`ALTER TABLE deployments ADD COLUMN origin_timestamp text  DEFAULT NOW() NOT NULL;`)
+  pgm.sql(`ALTER TABLE failed_deployments DROP COLUMN origin_server_url IF EXISTS;`)
```
Comment (agusaldasoro) [inline content/src/migrations/scripts/1620070748842_remove_origin.ts]:
```suggestion
  pgm.sql(
    `ALTER TABLE failed_deployments ADD COLUMN origin_server_url text  DEFAULT 'https://peer.decentraland.org/content' NOT NULL;`
  )
```

File: content/src/migrations/scripts/1620070748842_remove_origin.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  pgm.sql(`ALTER TABLE failed_deployments DROP COLUMN origin_timestamp IF EXISTS;`)
```
Comment (jmoguilevsky) [inline content/src/migrations/scripts/1620070748842_remove_origin.ts:23]:
```suggestion
  pgm.sql(`ALTER TABLE IF EXISTS failed_deployments DROP COLUMN origin_server_url;`)
  pgm.sql(`ALTER TABLEIF EXISTS  failed_deployments DROP COLUMN origin_timestamp;`)
```

Comment (agusaldasoro) [inline content/src/migrations/scripts/1620070748842_remove_origin.ts]:
```suggestion
  pgm.sql(`ALTER TABLE failed_deployments ADD COLUMN origin_timestamp text DEFAULT NOW() NOT NULL;`)
```

File: content/src/migrations/scripts/1620070748842_remove_origin.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+export async function down(pgm: MigrationBuilder): Promise<void> {
+  pgm.sql(`CREATE INDEX ON deployments ( origin_timestamp DESC );`) // Using plain SQL since lib doesn't expose DESC
+
+  pgm.sql(
+    `ALTER TABLE deployments ADD COLUMN origin_server_url text  DEFAULT 'https://peer.decentraland.org/content' NOT NULL;`
+  )
+  pgm.sql(`ALTER TABLE deployments ADD COLUMN origin_timestamp text  DEFAULT NOW() NOT NULL;`)
+  pgm.sql(`ALTER TABLE IF EXISTS failed_deployments DROP COLUMN origin_server_url;`)
+  pgm.sql(`ALTER TABLEIF EXISTS  failed_deployments DROP COLUMN origin_timestamp;`)
```
Comment (jmoguilevsky) [inline content/src/migrations/scripts/1620070748842_remove_origin.ts:23]:
```suggestion
  pgm.sql(`ALTER TABLE IF EXISTS  failed_deployments DROP COLUMN origin_timestamp;`)
```

**Review by agusaldasoro — COMMENTED** [review]
Great Work! Just check one comment

**Review by agusaldasoro — APPROVED** [review]

File: content/src/migrations/scripts/1620070748842_remove_origin.ts (hunk 2 above)
Comment (jmoguilevsky) [reply content/src/migrations/scripts/1620070748842_remove_origin.ts:8]: You're right, according to the documentation https://www.postgresql.org/docs/9.1/sql-altertable.html the drop table removes all indexes associated with the column, will remove the statement

**Pull request merged**